node/process_test.ts

```diff
@@ -342,39 +342,6 @@ Deno.test({
   },
 });
 
-// FIXME(bartlomieju):
-Deno.test({
-  name: "[process] stdio",
-  ignore: true,
-  async fn() {
-    const cwd = path.dirname(path.fromFileUrl(import.meta.url));
```

Comment:
This test seemed wrong anyway and we got a bunch of native node tests that cover it